pkg/build/builder/util_linux.go

```diff
@@ -13,7 +12,21 @@ import (
 // from the local /sys/fs/cgroup filesystem.  Overflow values are set to
 // math.MaxInt64.
 func GetCGroupLimits() (*s2iapi.CGroupLimits, error) {
-	byteLimit, err := readInt64("/sys/fs/cgroup/memory/memory.limit_in_bytes")
+	// see https://git.kernel.org/pub/scm/linux/kernel/git/tj/cgroup.git/tree/Documentation/admin-guide/cgroup-v2.rst
+	// for list of cgroupv2 files to try ... both support 'max' as a value in addition to int64
+	filesToTry := []string{"/sys/fs/cgroup/memory/memory.limit_in_bytes",
+		"/sys/fs/cgroup/memory.high",
```

Comment:
A quick spin through the crun and runc sources suggests that neither of them set `memory.high`, so we probably want to skip it.

Reply:
ok, which lines up with Guiseppe just suggesting `memory.max` in the bugzilla

Reply:
for completeness, both runc/crun supports setting `memory.high` through the unified map (podman exposes it with `--cgroup-conf`) but I don't think we should worry about it, as a cgroup is still allowed to allocate more memory until it reaches `memory.max`